pages/resources/skills.md

```diff
@@ -157,7 +157,7 @@ This list has the following categories:
 
 ### Other
   - Domain-Specific Knowledge
-    - Domains according to DFG.de (http://dfg.de/)
+    - Domains according to DFG.de (https://www.dfg.de/)
```

Comment:
Good we're switching to the https anyway but, aha, I hadn't noticed we were using the http, that's why I had difficulty reproducing it.

Reply:
Thanks!